src/components/report-technical-issue/template.njk

```diff
@@ -1,8 +1,14 @@
 {% set language %}{% if params.language == 'cy' %}cy{% else %}en{% endif %}{% endset %}
 {% set serviceId = params.serviceId or params.serviceCode %}
+{% set urlParametersBuilder %}
+  {% if params.referrerUrl or serviceId %}?{% endif %}
+  {% if serviceId %}service={{ serviceId | urlencode }}{% endif %}
+  {% if params.referrerUrl and serviceId %}&amp;{% endif %}
+  {% if params.referrerUrl %}referrerUrl={{ params.referrerUrl | urlencode }}{% endif %}
+{% endset %}
```

Comment:
and the output doesn't get any extra spaces / newlins in it? 🤯 (seems not from the tests) beautiful!